src/webrtc/call.ts

```diff
@@ -1663,7 +1663,8 @@ export function setVideoInput(deviceId: string) { videoInput = deviceId; }
 export function createNewMatrixCall(client: any, roomId: string, options?: CallOpts) {
     // typeof prevents Node from erroring on an undefined reference
     if (typeof(window) === 'undefined' || typeof(document) === 'undefined') {
-        logger.info("No window or document object: WebRTC is not supported in this environment");
+        // NB. We don't log here as apps try to create a call object as a test for
```

Comment:
I was going to suggest using `console.debug` to hide it in typical envs only to find out that `console.debug` is an alias for `console.log` in Node :'(

Reply:
Oh but we're not even using bare console, but we set up loglevel with DEBUG on - so either way no bueno